### ISSUE TITLE
Unicode chars (Umlaute) breaks cs_network module

### ISSUE THREAD

**Issue by profhase** [ansible/ansible-modules-extras#2511]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

cs_network
##### ANSIBLE VERSION

```
2.0.2.0
```
##### OS / ENVIRONMENT

ubuntu 14.04
##### SUMMARY

If there is a network with Umlaut in the description, the `cs_network` module breaks with

```
'ascii' codec can't encode cha
racter u'\xfc' in position 10: ordinal not in ran
ge(128)               
```
##### STEPS TO REPRODUCE

Try to create a network when there is a network avaliable with description 'mööp'
##### EXPECTED RESULTS

success
##### ACTUAL RESULTS

```
UnicodeEncodeError
```

**Comment by resmo** [comment]
Could you provide the traceback please?

**Comment by profhase** [comment]
> An exception occurred during task execution. To s
> ee the full traceback, use -vvv. The error was: U
> nicodeEncodeError: 'ascii' codec can't encode cha
> racter u'\xfc' in position 10: ordinal not in ran
> ge(128)  
> fatal: [localhost]: FAILED! => {"changed": false,
>  "failed": true, "module_stderr": "Traceback (mos
> t recent call last):\n  File \"/home/me/.ansibl
> e/tmp/ansible-tmp-1467366371.28-95175403218592/cs
> _network\", line 3002, in <module>\n    main()\n 
>  File \"/home/me/.ansible/tmp/ansible-tmp-14673
> 66371.28-95175403218592/cs_network\", line 1008, 
> in main\n    network = acs_network.present_networ
> k()\n  File \"/home/me/.ansible/tmp/ansible-tmp
> -1467366371.28-95175403218592/cs_network\", line 
> 854, in present_network\n    network = self.updat
> e_network(network)\n  File \"/home/me/.ansible/
> tmp/ansible-tmp-1467366371.28-95175403218592/cs_n
> etwork\", line 862, in update_network\n    if sel
> f._has_changed(args, network):\n  File \"/home/me/.ansible/tmp/ansible-tmp-1467366371.28-9517540
> 3218592/cs_network\", line 458, in _has_changed\n
>     return self.has_changed(want_dict=want_dict, 
> current_dict=current_dict, only_keys=only_keys)\n
>   File \"/home/me/.ansible/tmp/ansible-tmp-1467
> 366371.28-95175403218592/cs_network\", line 477, 
> in has_changed\n    elif str(value).lower() != st
> r(current_dict[key]).lower():\nUnicodeEncodeError
> : 'ascii' codec can't encode character u'\xfc' i
> n position 10: ordinal not in range(128)\n", "mod
> ule_stdout": "", "msg": "MODULE FAILURE", "parsed
> ": false}

**Comment by resmo** [comment]
Fix proposal https://github.com/ansible/ansible/pull/16601. Any chance to verify the fix?

**Comment by resmo** [comment]
Closing due https://github.com/ansible/ansible/pull/16601 has been merged and backported to 2.1.1
